frontend/src/components/VSearchResultsGrid/VAudioList.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <!-- Negative margin compensates for the `p-4` padding in row layout. -->
+  <ol
+    :aria-label="collectionLabel"
+    class="-mx-2 flex flex-col md:-mx-4"
+    :class="{ 'gap-4': isRelated }"
+  >
+    <li
+      v-for="audio in results"
+      :key="audio.id"
+      :class="{ 'mb-2 md:mb-1': !isRelated }"
```

Comment:
Nit: This can be accomplished with a different gap on line L6 using `isRelated ? 'gap-4' : 'gap-2 md:gap-1'`.